test/e2e/vertical_scaling_test.go

```diff
@@ -275,10 +278,25 @@ func ensureHealthyMember(t testing.TB, etcdClientFactory etcdClientCreator, memb
 	require.NoError(t, err)
 	defer closeFn()
 
-	// since we have a direct connection with the member
-	// getting any response is a good sign of healthiness
 	ctx, cancel := context.WithTimeout(context.TODO(), 15*time.Second)
 	defer cancel()
+
+	// // Learner and voting members both support the endpoint status call
+	// resp, err := etcdClient.Status(ctx, "https://127.0.0.1:2379")
```

Comment:
we call it after `ensureVotingMembersCount`, so it seems we don't need this distinction, right?

Reply:
Yep that's correct. Forgot to remove this block in the last update.